core/blockly.js

```diff
@@ -141,36 +141,27 @@ Blockly.svgSize = function(svg) {
 };
 
 /**
- * Schedule a call to the resize handler.  Groups of simultaneous events (e.g.
- * a tree of blocks being deleted) are merged into one call.
- * @param {Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ * Size the workspace when the contents change.  This also updates
+ * scrollbars accordingly.
+ * @param {!Blockly.WorkspaceSvg} workspace. Any workspace in teh SVG.
```

Comment:
"Any workspace in the SVG"--In whichSVG?  Rename to opt_workspace and add a note that if it is not specified, the main workspace will be used.